OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -144,7 +144,12 @@ void INotifyCreated.Created(Actor self)
 			conditionManager = self.Trait<ConditionManager>();
 		}
 
-		static int GetWeight(Actor a) { return a.Info.TraitInfo<PassengerInfo>().Weight; }
+		public int GetWeight(Actor a) { return GetPassengerTrait(a).Info.Weight; }
+
+		public Passenger GetPassengerTrait(Actor a)
```

Comment:
I would really prefer it if you fixed the code to properly support multiple passenger traits instead of writing a lint test.